app/views/layouts/mailer.html.haml

```diff
@@ -16,8 +16,8 @@
             %table{:bgcolor => "#182B3F"}
               %tr
                 %td
-                  %a{href: root_url}
-                    %img{:src => asset_url('logo-text-inside.png'), style: "max-width: 100px;"}/
+                  -# %a{href: root_url}
+                  %img{:src => asset_path('logo-text-inside.png'), style: "max-width: 100px;"}/
```

Comment:
I had to change this to asset_path to get it to render in development. I thought the action_mailer.default_url_options in development.rb should have made asset_url work but I may have misunderstood.